airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -201,8 +218,21 @@ private void assertCustomParametersDontOverwriteDefaultParameters(final Map<Stri
   public AirbyteMessageConsumer getConsumer(final JsonNode config,
                                             final ConfiguredAirbyteCatalog catalog,
                                             final Consumer<AirbyteMessage> outputRecordCollector) {
-    return JdbcBufferedConsumerFactory.create(outputRecordCollector, getDatabase(getDataSource(config)), sqlOperations, namingResolver, config,
-        catalog);
+    final DataSource dataSource = getDataSource(config);
+    if (TypingAndDedupingFlag.isDestinationV2()) {
```

Comment:
is this the best way to do this?

Reply:
TBD later on using Connector specific typed config